_posts/2023-09-07-develop-functions-locally.md

```diff
@@ -0,0 +1,278 @@
+---
+title: "The faster way to iterate on your OpenFaaS functions"
+description: "Learn how the local-run and watch features of faas-cli can help you iterate on functions faster."
```

Comment:
Somehow I think that description/first line is too technical. Can you change it to:

Learn how live reloading and local testing can help you iterate on functions faster with faas-cli.